content/projects/t2f-flex.md

```diff
@@ -0,0 +1,23 @@
+---
+title: T²F-Flex
+date: 2020-03-03
+publishdate: 2020-03-03
+weight: 10
+nav: false
+funding:
+- name: Bundesministerium für Bildung und Forschung, grant ID 031A262C
+  url: "#"
+people:
+- key: pratik.dhakal
+- key: michele.wichmann
+- key: dominique.sydow
+- key: robert.strothmann
+- key: andrea.volkamer
+---
+
+T²F-Flex extends the functionality of [T²F-Pharm](/projects/t2f-pharm) and offers pharmacophore modeling from MD simulations of apo structures, i.e. empty binding sites, to represent the dynamic nature of the structures.
+Interaction hotspots on the grid are calculated per snapshot (subset of the trajectory) and subsequently clustered and summarized into a single pharmacophore model. 
+This method allows to generate pharmacophores in the absence of ligand
+or interaction information and provides a feature-based description of protein binding sites.
+
```

Comment:
Add here 

`<!--more-->`

Details on Michele do not need to pop up here, right?
https://launch--admiring-blackwell-3e6756.netlify.app/research/structure-based/pharmacophores/

Reply:
Done!